### ISSUE TITLE
[Bug] 1.3 Seems to use target_outbound_peer_count for total target_peer_count.

### ISSUE THREAD

**Issue by scrutinously** [Chia-Network/chia-blockchain#10632]
### What happened?

Running a node on 1.3, my peer count is consistently low. My main farmer on 1.2.11 is always connected to max peers allowed by `target_peer_count`, but the 1.3 node will never go higher than the limit set by `target_outbound_peer_count`.

```
  target_outbound_peer_count: 6
  target_peer_count: 20
```
I confirmed this by changing the `target_outbound_peer_count` to multiple targets, and it always matched that target and would not exceed it.

### Version

1.3

### What platform are you using?

Linux

### What ui mode are you using?

CLI

**Comment by mariano54** [comment]
I think that might be related to upnp not working properly. Can you check that your port 8444 is accessible from outside?
https://portchecker.co/
https://github.com/Chia-Network/chia-blockchain/wiki/Resolving-Sync-Issues---Port-8444

**Comment by scrutinously** [comment]
Yeah, the port is open upnp is disabled, 8444 forwarded to my main farmer, somehow I still have outbound syncing from my 1.3 node despite 8444 forwarded to a different computer. Still, why is the node not attempting to get more connections when it should be looking for 14 inbound peers (20 peer count - 6 outbound count?).

**Comment by mariano54** [comment]
it will limit outbound to 6. It cannot obtain inbound, most likely because your computer is not accessible for some reason.

**Comment by scrutinously** [comment]
Shouldn't it be harder for outbound peer connections without UPNP and no port forwarding?

**Comment by mariano54** [comment]
Yeah that might be related..
